package.json

```diff
@@ -11,16 +10,18 @@
 		"postinstall": "patch-package"
 	},
 	"devDependencies": {
+		"@types/bn.js": "^5.1.1",
 		"@types/inquirer": "^9.0.3",
 		"patch-package": "^8.0.0",
 		"ts-node": "^10.9.2",
 		"typescript": "^5.0.2"
 	},
 	"dependencies": {
 		"@actions/core": "^1.10.1",
-		"@marinade.finance/marinade-ts-sdk": "^5.0.6",
+		"@coral-xyz/anchor": "^0.29.0",
 		"@solana/web3.js": "^1.78.4",
 		"@sqds/multisig": "^1.9.0",
+		"bn.js": "^5.2.1",
```

Comment:
No reason to add all the bloat of `@marinade.finance/marinade-ts-sdk` just for `bn.js`.